static/app/views/eventsV2/utils.tsx

```diff
@@ -92,6 +92,8 @@ export function decodeColumnOrder(fields: Readonly<Field[]>): TableColumn<string
     }
     column.column = col;
 
+    column.name = col.alias || column.name;
```

Comment:
We should always prefer `column.name` here, right now this is inverting behavior. If you are decoding a column with an alias and a name, the alias will be set to the name which should never be the case (or we need to extensively document this)